### PR TITLE
feat(medusa): add has_account to possible list_customers query params

### PR DESCRIPTION
**Pull request by pKorsholm**
**What**
- Add `has_account` as a possible query parameter for list-customers

### CODE REVIEW

File: packages/medusa/src/types/customers.ts [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 import { AddressPayload } from "./common"
 
 export class AdminListCustomerSelector {
   @IsString()
   @IsOptional()
   q?: string
 
+  @IsBoolean()
+  @IsOptional()
+  @Transform(({ value }) => value === "true")
```
Comment (olivermrbl) [inline packages/medusa/src/types/customers.ts:12]: todo: Use our custom boolean validator
```suggestion
  @Transform(({ value }) => optionalBooleanMapper.get(value))
```

**Review by olivermrbl — CHANGES_REQUESTED** [review]

**Review by adrien2p — APPROVED** [review]
LGTM 🎉

**Review by olivermrbl — APPROVED** [review]
LGTM!

**Pull request merged**